reflex/app.py

```diff
@@ -444,7 +444,7 @@ def compile(self):
         config = get_config()
 
         # Update models during hot reload.
-        if config.db_url is not None and not Model.automigrate():
+        if config.db_url is not None and config.db_migrate and not Model.migrate():
```

Comment:
Maybe we can wrap up this condition into a function - `if should_migrate():`

Reply:
this bit was removed entirely